koku/masu/database/trino_sql/reporting_awscostentrylineitem_summary_by_ec2_compute_p.sql

```diff
@@ -52,13 +74,13 @@ SELECT uuid() as uuid,
     memory,
     cast(
         map_filter(
-            cast(json_parse(tags) as map(varchar, varchar)),
+            cast(json_parse(tags[1]) as map(varchar, varchar)),
```

Comment:
I think we want all the tags 
```suggestion
            cast(json_parse(tags) as map(varchar, varchar)),
```